zipkin-web/src/main/scala/com/twitter/zipkin/web/Main.scala

```diff
@@ -53,7 +44,6 @@ trait ZipkinWebFactory { self: App =>
   )
 
   val webServerPort = flag("zipkin.web.port", new InetSocketAddress(8080), "Listening port for the zipkin web frontend")
-  val webRootUrl = flag("zipkin.web.rootUrl", "http://localhost:8080/", "Url where the service is located")
```

Comment:
Oh sorry, I didn't know that :-/ We should probably fix that (allow zipkin to ignore properties it hasn't heart of) so we don't deployment scripts...